Source/Scene/ShadowMap.js

```diff
@@ -149,6 +140,8 @@ define([
 
         this._enabled = defaultValue(options.enabled, true);
         this._softShadows = defaultValue(options.softShadows, false);
+        this._normalOffset = defaultValue(options.normalOffset, true);
+        this._normalOffsetScale = defaultValue(options.normalOffsetScale, 0.1);
```

Comment:
I would not expose this right now as an option or a getter/setter. The default values used before are tuned for the different bias modes.